debug/debug.go

```diff
@@ -291,7 +292,7 @@ func getPodDefinition(node string, namespace string, image string) *corev1.Pod {
 
 func createNamespace(cs *client.ClientSet, namespace string) error {
 	_, err := cs.Namespaces().Get(context.TODO(), namespace, metav1.GetOptions{})
-	if err != nil && !strings.Contains(err.Error(), "not found") {
```

Comment:
Can also be fixed in `if err != nil && !strings.Contains(err.Error(), "already exists") {` in this file?

Reply:
I changed this function completely by removing the first "get". I hope it's looks better now.